notebooks/274-Real-ESRGAN-image-restoration/274-Real-ESRGAN-image-restoration.ipynb

```diff
@@ -0,0 +1,740 @@
+{
```

Comment:
<p>Could you please provide also visualization of openvino results? for me, as user, it is important to see that result is the same like in torch and can be postprocessed using the same way, but I can not find this in your tutorial. Could you please add?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/939/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>